### ISSUE TITLE
Server installation ERR! EINTEGRITY

### ISSUE THREAD

**Issue by okbayou** [SignalK/signalk-server-node#829]
I tried to run the script to install on a raspberry 3: sudo npm install -g --unsafe-perm signalk-server
I would have attached the log file but couldn't find it.  (hidden files were enabled)
This is the result:

npm WARN registry Unexpected warning for https://registry.npmjs.org/: Miscellaneous Warning EINTEGRITY: sha512-4mq9KKjaxDQcgUlFClyrn1CNwgNRxmRqvGXfAF7ON6YU07MNNzGPlQ3QUCmT3CW2kTDaEjozFE9HVSGrGANkYQ== integrity checksum failed when using sha512: wanted sha512-4mq9KKjaxDQcgUlFClyrn1CNwgNRxmRqvGXfAF7ON6YU07MNNzGPlQ3QUCmT3CW2kTDaEjozFE9HVSGrGANkYQ== but got sha512-Jao2w62xjsNLdPaXaMevBV6DfqVhKMH5EZqbL9PWGomsntbGEu5Bi84CPF3mAhXjAw5KFIhimpckSapP6YHwGw==. (7159 bytes)
npm WARN registry Using stale data from https://registry.npmjs.org/ due to a request error during revalidation.
npm ERR! code EINTEGRITY
npm ERR! errno EINTEGRITY
npm ERR! Invalid response body while trying to fetch https://registry.npmjs.org/cookie: Integrity verification failed for sha512-4mq9KKjaxDQcgUlFClyrn1CNwgNRxmRqvGXfAF7ON6YU07MNNzGPlQ3QUCmT3CW2kTDaEjozFE9HVSGrGANkYQ== (/root/.npm/_cacache/content-v2/sha512/e2/6a/bd28a8dac4341c8149450a5cab9f508dc20351c6646abc65df005ece37a614d3b30d37318f950dd0502993dc25b69130da123a33144f475521ab18036461)

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-07-10T20_49_40_047Z-debug.log

**Comment by tkurki** [comment]
Please try again, it looks like a transient failure at npm.

**Comment by okbayou** [comment]
I tried it again with the same result.  This is a new install and I had trouble installing node-red until I found out that you had to run "sudo apt update --allow-releaseinfo-change" first due to some recennt changes.
I re ran this as well as update and upgrade and the signalK script still gives the same result.
I have installed signalK before on other boards with no problems.
Am I the only one with this problem?  If so I will start over.
Thanks,
Al

**Comment by sbender9** [comment]
Yeah, I think there may be an issue with your npm install.

I'd say don't install node-red. The node-red server plugin has it build in anyway.

-=Scott

**Comment by okbayou** [comment]
OK, I started with a new image on SD card and entered:  "sudo apt update --allow-releaseinfo-change" 
From that point I followed the **raspberry detailed instructions** for Signal K install and got an error because "npm" was missing, so I entered: "sudo apt install nodejs npm"
This(npm) should probably be added to the: raspberry_pi_installation.md
After that the install script ran OK!

I never did get anything after entering: "avahi-browse -a" so after waiting awhile I used Ctrl C to exit and continued the process of installing.

SignalK  appears to have installed properly and is running, so I will close this issue.

Thanks for the help!
Al
